app/src/containers/CoverPage.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux';
+import CoverPage from '../components/Home/CoverPage';
+import { getCoverPageEntries } from '../actions/coverPage';
+
+const mapStateToProps = (state) => ({
+  coverPageEntries: state.coverPageEntries
+});
```

Comment:
we have a reducer for this, which is used by the coverPage action, which is used by the coverPage component on load